packages/g/graphql-yoga.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "graphql-yoga",
+  "description": "Rewrite of a fully-featured GraphQL Server with focus on easy setup, performance & great developer experience. The core of Yoga implements WHATWG Fetch API and can run/deploy on any JS environment.",
+  "keywords": [
+    "nodejs",
+    "javascript",
+    "fetch",
+    "graphql",
+    "typescript",
+    "w3c",
+    "graphql-server",
+    "whatwg",
+    "bun",
+    "deno",
+    "the-guild"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/dotansimha/graphql-yoga",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dotansimha/graphql-yoga.git"
+  },
+  "authors": [
+    {
+      "name": "@dotan",
+      "email": "dotansimha@gmail.com"
+    },
+    {
+      "name": "@saihaj",
+      "email": "saihajpreet.singh@gmail.com"
+    }
+  ],
+  "autoupdate": {
+    "source": "npm",
+    "target": "graphql-yoga",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["*.js"]
```

Comment:
Looking at the `graphql-yoga` package on [NPM](https://www.npmjs.com/package/graphql-yoga?activeTab=code), there is no `dist` directory.

It looks like here are `cjs` and `esm` directories that contain code, but the `cjs` directory makes use of Node.js `require` calls that will not work in a browser, and the `esm` directory imports external modules which won't work unless module mapping is used afaik.

Is there a pre-bundled version that you should be attempting to reference here instead? Is this library meant to be used in a browser -- it looks like a server framework?